app/models/foreman_tasks/lock.rb

```diff
@@ -120,7 +120,7 @@ def link?(resource, uuid)
 
       # Records the information about the user that triggered the task
       def owner!(user, uuid)
-        build_owner(user, uuid).save!
```

Comment:
Any reason to keep this around https://github.com/theforeman/foreman-tasks/blob/4aa960623bfb19dff930b96efcf23bab7019a90b/lib/foreman_tasks/dynflow/persistence.rb#L39 ?

Reply:
I can't think of any, just an oversight on my side